nan_json.h

```diff
@@ -77,15 +77,19 @@ class JSON {
 #if NAN_JSON_H_NEED_PARSE
     return scope.Escape(parse(json_string));
 #else
+    Nan::MaybeLocal<v8::Value> result;
+#if NODE_MODULE_VERSION == 14  // v0.12
```

Comment:
@bnoordhuis / @kkoopa : You can cherry-pick it from me if you like: https://github.com/mkrufky/nan/commit/8d29b5691abb968d7b010dae131766e20330d3a7